docs/source/primitives.rst

```diff
@@ -314,18 +314,23 @@ Actions
 ..     }
 
 
+.. _data_display:
```

Comment:
Is this missing something ?

Reply:
this is the format of adding arbitrary references, so that they can be linked to later in the file =)